src/components/ui/Counter.tsx

```diff
@@ -0,0 +1,20 @@
+import classnames, {
+  fontSize,
+  textColor,
+  whitespace,
+} from 'classnames/tailwind'
+
+const counterText = (error?: boolean) =>
+  classnames(
+    textColor(error ? 'text-error' : 'text-formal-accent'),
```

Comment:
`text-formal-accent` should be default (if not, please set it inside `index.css` like : `html { textColor: var(--formal-accent) }`
So, here you can do:
```TS
textColor({'text-error': error})
```

Reply:
ty